homework_7_261122/src/cars/Car.java

```diff
@@ -0,0 +1,76 @@
+package cars;
+
+public class Car {
+    int maxSpeed;
+    int fuelAmount;
+    int productionYear;
+    double price;
+    String brand;
+    int currentSpeed;
+    boolean isEngineStarted = false;
+
+    public Car(int maxSpeed, int fuelAmount, int productionYear, double price, String brand) {
+        this.maxSpeed = maxSpeed;
+        this.fuelAmount = fuelAmount;
+        this.productionYear = productionYear;
+        this.price = price;
+        this.brand = brand;
+    }
+
+    public void getMaxSpeed() {
+        System.out.println("Max speed: " + maxSpeed);
+    }
+
+    public void getFuelAmount() {
+        System.out.println("Fuel amount: " + fuelAmount);
+    }
+
+    public void getProductionYear() {
+        System.out.println("Production year: " + productionYear);
+    }
+
+    public void getPrice() {
+        System.out.println("Price: " + price);
+    }
+
+    public void getBrand() {
+        System.out.println("Brand: " + brand);
+    }
+
+    public void setCurrentSpeed(int currentSpeed) {
+        this.currentSpeed = currentSpeed;
+    }
+
+    public void getCurrentSpeed() {
+        System.out.println("Current speed: " + currentSpeed);
+    }
+
+    public void getIsEngineStarted() {
+        System.out.println("Is Engine Started: " + isEngineStarted);
+    }
+
+    public void startStopEngine() {
```

Comment:
by logic of this method you either start up or shut down engine, better separate it in two methods for start up and shutdown